docs/06-contributors/999-rfcs/2022-05-28-winglang-reqs.md

```diff
@@ -6,7 +6,7 @@ description: The original requirements documents for the wing language experienc
 - **Author(s):**: @eladb
 - **Submission Date**: {2022-05-28}
 - **Stage**: Approved
-- **Implementation**: [Language Specification](https://github.com/winglang/wing/blob/main/docs/04-reference/winglang-spec.md)
+- **Implementation**: Language Specification
```

Comment:
Instead of removing it, let's fix the broken link:
```suggestion
- **Implementation**: [Language Specification](https://docs.winglang.io/reference/spec)
```